command/acl/acl_helpers.go

```diff
@@ -71,23 +72,26 @@ func GetPolicyIDFromPartial(client *api.Client, partialID string) (string, error
 	return policyID, nil
 }
 
-func GetPolicyIDByName(client *api.Client, name string) (string, error) {
+func GetPolicyByName(client *api.Client, name string) (*api.ACLPolicy, error) {
 	if name == "" {
-		return "", fmt.Errorf("No name specified")
+		return nil, fmt.Errorf("No name specified")
 	}
 
-	policies, _, err := client.ACL().PolicyList(nil)
+	policy, _, err := client.ACL().PolicyReadByName(name, nil)
 	if err != nil {
-		return "", err
+		return nil, errors.Wrapf(err, "failed to find policy with name %s", name)
```

Comment:
I encourage the use of `fmt.Errorf` with `%w` to wrap errors, so we don't have to import `github.com/pkg/errors`

Reply:
Done